src/main/webapp/resources/js/pages/dashboard/components/RecentActivityList.jsx

```diff
@@ -0,0 +1,55 @@
+import { Button, List, Space, Typography } from "antd";
+import React from "react";
+import { ActivityListItem } from "../../../components/activities/ActivityListItem";
+import { BORDERED_LIGHT } from "../../../styles/borders";
+
+/**
+ * Component to display list of recent activity
+ * @param activities List of recent activities
+ * @param total The total number of recent activities
+ * @param page The last loaded page of activities
+ * @param {function} setPage Function to set the page number
+ * @param loading Whether the data has finished loading or not
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function RecentActivityList({
+  activities,
+  total,
+  page,
+  setPage,
+  loading,
+}) {
+  return (
+    <Space direction={"vertical"} style={{ width: `100%` }}>
```

Comment:
Same comment here about the use of `Row`.  Prevents needing to add the `style={{width: "100%"}}` which I always thought was strange to add.

Reply:
Updated in 0042a0ad2e7a59b91ddeb0ff9efe5adaa46c6c25. Yeah I found that to be strange as well